docs/iris/example_code/Meteorology/inset_plot.py

```diff
@@ -0,0 +1,73 @@
+"""
+Test Data Showing Inset Plots
+=============================
+
+This example demonstrates the use of a single 3D data cube with time, latitude
+and longitude dimensions to plot a latitude-averaged temperature series with
+an inset plot of the data region.
+
+"""
+
+###############################################################################
+
+import matplotlib.pyplot as plt
+import numpy as np
+import iris
+import cartopy.crs as ccrs
+import iris.quickplot as qplt
+import iris.plot as iplt
+
+###############################################################################
+
+
+def main():
+    iris.FUTURE.netcdf_promote = True
```

Comment:
This would be better using the context manager form:

```python
with iris.FUTURE.context(netcdf_promote=True):
    cube1 = iris.load_cube(iris.sample_data_path('ostia_monsthly.nc'))
```